### ISSUE TITLE
ebmbot could do with a command to restart the application

### ISSUE THREAD

**Issue by evansd** [ebmdatalab/openprescribing#2392]
There's a specific need for this after an import in order to pick up the new matrixstore file. But I can also imagine it being useful in cases where something has got in a weird state and none of us have easy access to an SSH session.

**Comment by evansd** [comment]
We actually still don't have this and I still think it would be useful!

**Comment by richiecroker** [comment]
@evansd fair enough, is this an _openprescribing_ issue though, or should it be somewhere else?
